Directory.Build.props

```diff
@@ -9,11 +9,12 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <HighEntropyVA>true</HighEntropyVA>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>latest</LangVersion>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Product>Microsoft Health</Product>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <SupportedFrameworks>netcoreapp3.1;net5.0</SupportedFrameworks>
+    <SdkPackageVersion>6.0.0</SdkPackageVersion>
+    <SupportedFrameworks>net5.0;net6.0</SupportedFrameworks>
```

Comment:
>>net5.0;net6.0 [](http://example.com/codeflow?start=24&length=14)

.net 3 and .net 6 are Long term support, while .net5 is not.
[https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle](https://dotnet.microsoft.com/platform/support/policy/dotnet-core#lifecycle)

I would throw away .net 5 and leave support for .net 3 and .net 6.

Reply:
If both FHIR and DICOM both don't have any libraries still running `netcoreapp3.1`, I'm inclined to remove it. But I think we need to keep `net5.0` in case migrations to `net6.0` take a bit. Is there any FHIR code that needs `netcoreapp3.1`?

Reply:
No, we don't and you right it's make sense to have one LTS version and previous one rather than two LTS